app/components/edit-reminder.js

```diff
@@ -24,6 +24,11 @@ export default Ember.Component.extend({
         record.save();
       })
     },
+    reset(){
+      this.element.children[0].textContent = this.model.data.title;
```

Comment:
@martensonbj doesn't that only work if you create dirty data? i never actually touch the actual data so nothing is ever dirty.